packages/beacon-node/src/api/impl/errors.ts

```diff
@@ -35,3 +35,17 @@ export class OnlySupportedByDVT extends ApiError {
     super(501, "Only supported by distributed validator middleware clients");
   }
 }
+
+export class MultipleError extends ApiError {
```

Comment:
We could use the same name as in the spec, see [types/http.yaml#L152](https://github.com/ethereum/beacon-APIs/blob/0fc77b79df7e6293192cf4cd97a8d94387ea640d/types/http.yaml#L152)

```suggestion
export class IndexedError extends ApiError {
```